clients/ts/FunctionalTests/package.json

```diff
@@ -39,7 +39,7 @@
     "build:lint": "node ../common/node_modules/tslint/bin/tslint -c ../tslint.json -p ./tsconfig.json",
     "build:webpack": "node ../common/node_modules/webpack-cli/bin/cli.js",
     "build:parent": "cd .. && npm run build",
-    "pretest": "npm run build:parent && npm run build && dotnet build",
+    "pretest": "npm run build:parent && npm run build && dotnet build --no-restore",
     "test": "npm run test:local --",
     "test:inner": "npm run build && dotnet build && npm run test:local --",
```

Comment:
FYI this one is used when devs are making changes, it's not run by `build.cmd`, so I don't think it's needs to change.